bodhi/server/util.py

```diff
@@ -81,53 +91,119 @@ def get_rpm_header(nvr, tries=0):
 
 
 def get_nvr(nvr):
-    """ Return the [ name, version, release ] a given name-ver-rel. """
+    """
+    Return the (name, version, release) from a given name-ver-rel string.
+
+    Args:
+        nvr (basestring): The name-version-release string that you wish to know the split nvr from.
```

Comment:
Same comment about the basestring as above. I see it's used throughout, so I'll stop pointing them out.